container/openejb-loader/src/main/java/org/apache/openejb/loader/Files.java

```diff
@@ -245,8 +356,8 @@ public static File mkdirs(final File file) {
     }
 
 
-    // Shutdown hook for recursive delete on tmp directories
-    static final List<String> delete = new ArrayList<String>();
+    // Shutdown hook for recursive DELETE on tmp directories
+    static final List<String> DELETE = new ArrayList<String>();
```

Comment:
diamond :)

Reply:
Could you check if this variable can be `private`, I'm not sure

Reply:
absolutely